flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/api/bridge/java/StreamTableEnvironment.java

```diff
@@ -788,61 +788,6 @@ DataStream<Row> toChangelogStream(
      *     be used as source of truth.
      */
     @Deprecated
-    <T> DataStream<T> toAppendStream(Table table, Class<T> clazz);
-
-    /**
-     * Converts the given {@link Table} into an append {@link DataStream} of a specified type.
-     *
-     * <p>The {@link Table} must only have insert (append) changes. If the {@link Table} is also
-     * modified by update or delete changes, the conversion will fail.
-     *
-     * <p>The fields of the {@link Table} are mapped to {@link DataStream} fields as follows:
-     *
-     * <ul>
-     *   <li>{@link Row} and {@link org.apache.flink.api.java.tuple.Tuple} types: Fields are mapped
-     *       by position, field types must match.
-     *   <li>POJO {@link DataStream} types: Fields are mapped by field name, field types must match.
-     * </ul>
-     *
-     * @param table The {@link Table} to convert.
-     * @param typeInfo The {@link TypeInformation} that specifies the type of the {@link
-     *     DataStream}.
-     * @param <T> The type of the resulting {@link DataStream}.
-     * @return The converted {@link DataStream}.
-     * @deprecated Use {@link #toDataStream(Table, Class)} instead. It integrates with the new type
-     *     system and supports all kinds of {@link DataTypes} that the table runtime can produce.
-     *     The semantics might be slightly different for raw and structured types. Use {@code
-     *     toDataStream(DataTypes.of(TypeInformation.of(Class)))} if {@link TypeInformation} should
-     *     be used as source of truth.
-     */
-    @Deprecated
-    <T> DataStream<T> toAppendStream(Table table, TypeInformation<T> typeInfo);
-
-    /**
-     * Converts the given {@link Table} into a {@link DataStream} of add and retract messages. The
```

Comment:
You removed the comment for toRetractStream but kept the comment for toAppendStream.